src/modules/lite/explorer/components/VoteDetails.tsx

```diff
@@ -218,17 +220,20 @@ export const VoteDetails: React.FC<{
             )}
             {getTotalVoters(choices) > 0 ? (
               <DownloadCsvFile
-                data={choices}
+                data={groupedVotes}
                 pollId={poll?._id}
+                decimals={tokenData?.decimals ? tokenData?.decimals : ""}
+                isXTZ={isXTZ}
                 symbol={isXTZ ? "XTZ" : tokenData?.symbol ? tokenData?.symbol : ""}
               />
             ) : null}
           </Grid>
         </LegendContainer>
         <VotesDialog
-          decimals={tokenData?.decimals ? tokenData?.decimals : ""}
+          decimals={tokenData?.decimals ? tokenData?.decimals : "6"}
```

Comment:
I'm not sure if it's the best idea to default the decimal to 6, I'm not sure what we can do here but decimals = 6 should only be the case with isXTZ